test/jitter/mem_breakpoint.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import print_function
+import sys
+
+from miasm.core.utils import decode_hex
+from miasm.analysis.machine import Machine
+from miasm.jitter.csts import PAGE_READ, PAGE_WRITE, \
+    EXCEPT_BREAKPOINT_MEMORY, EXCEPT_ACCESS_VIOL
+
+def mem_breakpoint_handler(jitter):
+    print("======")
+    print("Data access caught!")
+
+    mem_r = jitter.vm.get_memory_read()
+    if len(mem_r) > 0:
+        for s, e in mem_r:
+            print("%s - %s" % (hex(s), hex(e - s)))
+    else:
+        print("No read")
+
+    mem_w = jitter.vm.get_memory_write()
+    if len(mem_w) > 0:
+        for s, e in mem_w:
+            print("%s - %s" % (hex(s), hex(e - s)))
+    else:
+        print("No write")
+
+    print("pc = %s" % (hex(jitter.cpu.PC)))
+    print("[DBG] vm.exception = %d" % (jitter.vm.get_exception()))
+    print("======")
+
+    # Cleanup
+    jitter.vm.set_exception(0)
+    jitter.vm.reset_memory_access()
+
+    return True
+
+machine = Machine("aarch64l")
+jitter = machine.jitter(sys.argv[1])
+
+start_addr = 0xFFFFFF8008080000
+end_addr = start_addr + 0x8000000
```

Comment:
@idl3r : to keep the code simple, maybe we can add directly a big page instead of the loop adding page per page (in Miasm, there is no need to have physically aligned/sized memory pages ).
for example:
```python
itter.vm.add_memory_page(start_addr, PAGE_READ|PAGE_WRITE, b"\x00"*(end_addr-start_addr), "code page")
```
Are you ok with this?